LICENSE.md

```diff
@@ -1,4 +1,4 @@
-mapbox-gl-native-ios Copyright (c) 2014-2020 Mapbox.
+mapbox-gl-native-ios copyright (c) 2014-2020 Mapbox.
```

Comment:
Really tempted to correct that ASCII `(c)` and `-` into something typographically correct…